README.md

```diff
@@ -0,0 +1,32 @@
+# How TypeScript Community Works
+
+TypeScript is a discord community for people interested in TypeScript
+
+It's ran by:
+ 
+ - [dream](https://github.com/Draem)
+ - [Haxified](https://github.com/Haxified)
+ 
+With help moderating by:
+
+ - [Robinlemon](https://github.com/Robinlemon)
+ - [Orta Therox](https://github.com/orta)
+
+## How to join
+
+[Click here](https://discord.gg/rqAeTR5) to get auto-invited. You will need a Discord account.
+
+## Community Rules
+
+- Follow the [Code of Conduct](./CODE_OF_CONDUCT.md)
+- Harassment or threats will not be tolerated.
+- Prefer English in channels which are not bilingual.
+- Do not spam, troll or advertise in the server or the DMs of members.
+- Do not post NSFW content.
+- Please keep your language clean.
+- Use channels to help keep focus, keep everything in its correct channel.
+- Always follow Discord's TOS.
+- Staff always have the final decision.
+- Use common sense and have fun.
```

Comment:
I re-framed some of these to be positives instead of negatives (e.g. _do x_, not _don't do x_)